README.md

```diff
@@ -90,7 +90,7 @@ cd tango
 Then run
 
 ```bash
-pip install -e .
+pip install -e .[all]
```

Comment:
I think most people installing from source will want this, but I can revert this change if you disagree.

Reply:
I would leave this as-is since the next sentence below shows how to install with integrations, but I think it would be worth adding a note about the "all" integration.

Reply:
Do you disagree that most people would want `[all]` if they are installing from source?

Reply:
No, but if we're going to change this line we should also update the lines below to explain how to install locally with only specific integrations or no integrations.

Reply:
I think I did exactly what you suggested already.  The next explanation is how to install a specific integration and then it explains how to install just the base library.

Reply:
Oh, somehow I missed that. Sorry!